README.md

```diff
@@ -24,6 +24,14 @@ Install bash shell completions with:
 sectionctl install-completions
 ```
 
+### Using `sectionctl` in CI/CD
+
+You can set credentials via the `SECTION_USERNAME` and `SECTION_TOKEN` environment variables:
+
+``` bash
+SECTION_USERNAME=me@domain.example SECTION_TOKEN=s3cr3t sectionctl accounts list
```

Comment:
Often interactive bash shells have `HISTCONTROL=ignorespace` [or equivalent](https://www.gnu.org/software/bash/manual/html_node/Bash-Variables.html#index-HISTCONTROL) defined in the default profile, which means that using a leading space on a command like this will prevent it from being logged in the bash history file with the token. Not sure if or how to communicate this to a user.